test/__snapshots__/StatsTestCases.test.js.snap

```diff
@@ -8,9 +8,9 @@ Child fitting:
     Built at: Thu Jan 01 1970 00:00:00 GMT
                       Asset      Size  Chunks             Chunk Names
     9ac13fb7087e9ff1b93e.js  1.05 KiB       0  [emitted]  
-    f2e891598128a57b072c.js    11 KiB       1  [emitted]  
+    f2e891598128a57b072c.js  11.6 KiB       1  [emitted]  
```

Comment:
bundle size of most of these test cases shouldn't increase, since prefetching isn't used. We don't want to users to pay for not using this feature.

Reply:
Following most of the other comments, I was able to eliminate most of the differences. However, to not emit blank lines when not using prefetching, I followed a different pattern than the rest of the code:

```js
"var moreModules = data[1];",
withDefer ? "var executeModules = data[2];" : "",
...(withPrefetch ? ["var prefetchChunks = data[3] || []"] : []),
'// add "moreModules" to the modules object,',
```